src/Client.php

```diff
@@ -99,62 +98,40 @@ private function handleRequestException(GuzzleException $e)
         }
     }
 
-    /**
-     * @param EncryptRequest $encryptRequest
-     * @param $keyName
-     * @param $keyVersion
-     * @return KeyOperationResult
-     */
-    public function encrypt(EncryptRequest $encryptRequest, $keyName, $keyVersion)
+    public function encrypt(EncryptRequest $encryptRequest, string $keyName, string $keyVersion): KeyOperationResult
     {
         $request = new Request(
             'POST',
             sprintf('keys/%s/%s/encrypt?api-version=%s', $keyName, $keyVersion, self::API_VERSION),
             [],
-            \GuzzleHttp\json_encode($encryptRequest->getArray())
+            (string) json_encode($encryptRequest->getArray())
```

Comment:
JSON_THROW_EXCEPTION?

Reply:
no tak je to encode na getArray, ale jo pro zichr :)